.github/workflows/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEAT] "
+labels: enhancement
+assignees: ''
+
+---
+
+## Feature Description
+
+**Summary:**
+Describe the feature clearly and concisely.
+
+## Proposed Method
+
+**Summary:**
+Describe the suggested method of implementation clearly and concisely.

```

Comment:
I don't think there's a big need for the "**Summary**" part, "Feature Description"/"Proposed Method" should be more than enough.